units/commit_queue.go

```diff
@@ -543,20 +551,6 @@ func checkPR(ctx context.Context, githubToken, issue, owner, repo string) (*gith
 		return nil, true, errors.Wrap(err, "GitHub returned an incomplete PR")
 	}
 
-	if pr.Mergeable == nil {
```

Comment:
I think this caused some weird problem I couldn't reproduce where a pr got stuck. They aren't needed anyway since git.merge_pr checks them